src/nft-card.ts

```diff
@@ -190,24 +189,37 @@ export class NftCard extends LitElement {
               .traitData=${this.traitData}
             ></nft-card-back>
             `
-          }
+        }
        </div>
      `
-   }
-
-  /**
-	  * async connectWallet - Initializes connection to the injected web3 account
-	  */
-  private async connectWallet() {
-    if (!window.web3) {
-      //
-    } else if (window.ethereum) {
-      window.ethereum.enable()
-    } else {
-      const errorMessage =
-        'You need an Ethereum wallet to interact with this marketplace. Unlock your wallet, get MetaMask.io or Portis on desktop, or get Trust Wallet or Coinbase Wallet on mobile.'
-      alert(errorMessage)
-      throw new Error(errorMessage)
     }
-  }
+
+    private async getAsset(contractAddress, tokenId) {
+        this.asset = await this.seaport.api.getAsset(contractAddress, tokenId);
+    }
+
+    private getNetwork() {
```

Comment:
this doesn't look used in this diff (maybe somewhere else) but it also looks like it's just returning the value of `this.network`,  cast as a Network

Reply:
I'm still confused about this. even for mainnet, "main" is the same as `Network.Main`

Reply:
Yea this is probably just baggage I'll remove